platforms/documentation/docs/src/docs/userguide/best-practices/bp_general.adoc

```diff
@@ -15,6 +15,29 @@
 [[best_practices_general]]
 = General Gradle Best Practices
 
+[[use_kotlin_dsl]]
+== Use Kotlin DSL
+
+Prefer the Kotlin DSL (`build.gradle.kts`) over the traditional Groovy DSL (`build.gradle`) when authoring new builds or creating new subprojects in existing builds.
```

Comment:
Do we want to call it "traditional"? It kind of implies default. Maybe we just say groovy DSL?